src/app.go

```diff
@@ -66,6 +67,11 @@ func New(cfg *Config, logger log.Logger) (*app.App, error) {
 		return nil, err
 	}
 
+	err = aliases.RegisterService(a, postgresClient)
```

Comment:
Seems you can skip registering service in this PR as you only implemented the data layer.

Reply:
yes, [I agree](https://github.com/ConsenSys/quorum-key-manager/pull/214#discussion_r679090238)